tests/RankedPairsCalculatorTest.php

```diff
@@ -234,4 +234,92 @@ public function testPairTieBreakingWitTiedBallot() : void
         $expectedWinners = new CandidateList($this->claire, $this->bob, $this->alice);
         $this->assertEquals($expectedWinners, $actualWinners->getRanking());
     }
+
+    public function testTideman1987Example5()
+    {
+        $expectedRanking = (new CandidateRankingParser())->parse("V>W>X>Y>Z");
+
+        $ballots = (new TestScenarioTideman1987Example2())->getBallots();
```

Comment:
If you look at these TestScenario classes you will see a lot of ugly generated code representing ballots. In a future pull request I would like to update the TestScenario classes to use the NBallot syntax so that we can understand the ballots cast in our test elections at a glance.